app/server/api-generator/src/main/java/io/syndesis/server/api/generator/soap/parser/XmlSchemaExtractor.java

```diff
@@ -224,6 +227,14 @@ public XmlSchema getOrCreateTargetSchema(String namespaceURI) {
         XmlSchema targetSchema = targetSchemas.getSchemaByTargetNamespace(namespaceURI);
         if (targetSchema == null) {
             targetSchema = targetSchemas.newXmlSchemaInCollection(namespaceURI);
+            final NamespaceMap namespaceContext = new NamespaceMap();
+
+            // add the usual suspects, for some reason these seem to be missing from the
+            // generated SchemaSet
+            namespaceContext.add("xsd", XMLConstants.W3C_XML_SCHEMA_NS_URI);
+            namespaceContext.add(XMLConstants.XML_NS_PREFIX, XMLConstants.XML_NS_URI);
+
+            targetSchema.setNamespaceContext(namespaceContext);
```

Comment:
Could the ns context from source schema be copied, in case there are other properties defined there?

Reply:
This is the default context, added to each target schema set, additional namespaces are added to the context as needed. These were missing, especially difficult one is the XML namespace which seems implicitly defined in some cases only.